lib/src/base-styles/_nx-btn.scss

```diff
@@ -101,6 +101,17 @@
       height: 16px;
     }
   }
+
+  &--small {
+    font-size: var(--nx-font-size-s);
+    padding: var(--nx-spacing-base) var(--nx-spacing-3x);
+    margin: 0 var(--nx-spacing-3x) 0 0;
+
+    .nx-icon {
+      height: var(--nx-spacing-3x);
+      margin-right: var(--nx-spacing-2x);
+    }
+  }
```

Comment:
The colors look fine to me but are slightly different to figma e.g.

Figma
![image](https://github.com/user-attachments/assets/335bdac0-23f4-41e4-ac7a-55b230ee4ea1)

Actual
![image](https://github.com/user-attachments/assets/b30348d9-39a7-45c3-9fa7-67a2488c5bdb)

and

Figma
![image](https://github.com/user-attachments/assets/7e0fc0c8-3bd3-478b-93fa-684abcf594e9)


Actual
![image](https://github.com/user-attachments/assets/e61442ce-08a9-4854-a6c0-0de8a2e0cd04)

Reply:
updated the style to align with design